backend/src/nodes/model_save_nodes.py

```diff
@@ -32,11 +33,14 @@ def __init__(self):
 
         self.side_effects = True
 
-    def run(self, onnx_model: bytes, directory: str, model_name: str) -> None:
+    def run(
+        self, onnx_model: Tuple[Any, bytes], directory: str, model_name: str
```

Comment:
How is this correct? The `OnnxLoadModelNode` nodes returns `bytes`, so why is this suddenly `Tuple[Any, bytes]`?

Reply:
It does not, anymore 

https://github.com/joeyballentine/chaiNNer/blob/00cde1256fa1d07f0c3cfa012a8a7bbec95c5ee5/backend/src/nodes/onnx_nodes.py#L106

You must be looking at outdated code

Reply:
Indeed, I was.